.github/workflows/push-release.yml

```diff
@@ -445,26 +449,31 @@ jobs:
       - name: Create dist folder
         run: mkdir -p dist
       - name: Setup ecm-distro-tools
-        uses: rancher/ecm-distro-tools@v0.36.0
+        uses: rancher/ecm-distro-tools@v0.38.1
         with:
-          version: v0.36.0
-      - name: Generate Linux Digests
+          version: v0.38.1
+      - name: Create ECM Config
         run: |
           mkdir -p ~/.ecm-distro-tools
           release config gen
+          mkdir -p /tmp/digests
+      - name: Generate Linux Digests
+        run: |
+          export LINUX_IMAGES_URL=https://github.com/${{ github.repository_owner }}/rancher/releases/download/${{ env.TAG}}/rancher-images.txt
           release generate rancher docker-images-digests \
-            --artifact-url "https://github.com/${{ env.REPOSITORY_OWNER }}/rancher/releases/download/${{ env.TAG }}/rancher-images.txt" \
-            --output-file ./dist/rancher-images-digests-linux-amd64.txt \
-            --registry "docker.io"
-          cp ./dist/rancher-images-digests-linux-amd64.txt ./dist/rancher-images-digests-linux-arm64.txt
+            --images-url "$LINUX_IMAGES_URL" \
+            --output-file /tmp/digests/rancher-images-digests-linux-amd64.txt \
+            --registry ${{ env.DOCKER_REGISTRY }}
+          cp /tmp/digests/rancher-images-digests-linux-amd64.txt /tmp/digests/rancher-images-digests-linux-arm64.txt
       - name: Generate Windows Digests
         run: |
+          export LINUX_IMAGES_URL=https://github.com/${{ github.repository_owner }}/rancher/releases/download/${{ env.TAG}}/rancher-images.txt
```

Comment:
JFYI that in theory `${{ github.repository_owner }}/rancher` could be changed to `${{ github.repository }}`.